### PR TITLE
don't load wallets at login

### PR DESCRIPTION
**Pull request by zanderz**
is it as simple as this?

### CODE REVIEW

**Comment by chrisnojima** [comment]
@buoyad ^

**Comment by buoyad** [comment]
@keybase/react-hackers this is ready!

File: shared/actions/wallets.js [hunk 1]
```diff
@@ -14,13 +14,14 @@ import {getPath} from '../route-tree'
...
 import {getEngine} from '../engine'
+import {anyWaiting} from '../constants/waiting'
 
 const buildPayment = (state: TypedState, action: any) =>
   RPCTypes.localBuildPaymentLocalRpcPromise({
     amount: state.wallets.buildingPayment.amount,
     // FIXME: Assumes XLM.
-    fromPrimaryAccount: !state.wallets.buildingPayment.from,
-    from: state.wallets.buildingPayment.from,
+    fromPrimaryAccount: state.wallets.buildingPayment.from === Types.noAccountID,
```
Comment (zanderz) [inline shared/actions/wallets.js]: this was my change, based on your suggestion, but should it now be `state.wallets.builtPayment`?

Comment (buoyad) [reply shared/actions/wallets.js]: `builtPayment` will always be empty when the send form is first opened, though. the idea is to pass the input from the form to `BuildPaymentLocal` and then use the resolved fields in `SendPaymentLocal`. In this case in `BuildPaymentLocal` the form says `fromPrimaryAccount: true`, we get the address which we put into `builtPayment.from`, which we then send off to `SendPaymentLocal`

**Comment by zanderz** [comment]
it won't let me click the review button 'cause I'm the original author, but - lgtm

**Pull request merged**